tests/backup/backup_basic_test.go

```diff
@@ -284,6 +290,7 @@ var _ = AfterSuite(func() {
 
 func TestMain(m *testing.M) {
 	// call flag.Parse() here if TestMain uses flags
+	log.Infof("Snigdha: This is the start of the script")
```

Comment:
Do you really need this it looks like a debug log line?

Reply:
removed